lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -30,14 +30,10 @@ extension BridgedSourceLoc {
   /// Form a source location at the given absolute position in `buffer`.
   init(
     at position: AbsolutePosition,
-    in buffer: UnsafeBufferPointer<UInt8>
+    in buffer: UnsafeBufferPointer<UInt8>?
```

Comment:
There’s no reason to change the method signature. All that matters, was unwrapping  the `baseAddress` of `buffer`. Also, instead of copy-pasting code in the comments, just push, that makes it a lot easier to review.

Reply:
> There’s no reason to change the method signature

the method signature needs to be changed because it says I `Cannot force unwrap value of non-optional type`. 

> copy-pasting code in the comments

I usually do this when we are having a discussion about the direction to go. If I am making changes on my own, I push them up. I can see why it makes things easier to review and can start pushing more often.

Reply:
Posted after it was already resolved in the other thread but:

> buffer.baseAddress is the optional here, so @AnthonyLatsis just meant buffer.baseAddress!